core/auth/src/test/java/software/amazon/awssdk/auth/credentials/DefaultCredentialsProviderTest.java

```diff
@@ -17,9 +17,15 @@
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
+import com.google.common.jimfs.Jimfs;
+import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.nio.file.Path;
 import java.util.Arrays;
 import java.util.List;
 import java.util.function.Supplier;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
These imports were added but no code was added?

Reply:
Ah good catch - I've removed! Yeah, I had originally implemented some tests here, but ended up moving them into ProfileCredentialsProviderTest and the DefaultClientBuilderTest instead since they better matched the implementation.